pkg/kudoctl/cmd/install_test.go

```diff
@@ -49,26 +49,3 @@ func TestTableNewInstallCmd_WithParameters(t *testing.T) {
 		assert.NotNil(t, err, test.errorMessage)
 	}
 }
-
-var parameterParsingTests = []struct {
```

Comment:
these tests disappeared... shouldn't they move to `params_test.go` ?

Reply:
it got moved to params_test. Oh wait, I did not push that, LOL :D